### DISCUSSION TITLE
[BUG] mobile video player

### DISCUSSION THREAD

**Discussion by maaxpane** [immich-app/immich#4765] [Feature Request]
### The bug

When accessing immich through the Chrome browser on Android and watching a video, you do not have the option to click on a certain time in the video, on the web it works normally and it is possible to click on any minute of the video

### The OS that Immich Server is running on

ubuntu 22.04

### Version of Immich Server

v1.79.1

### Version of Immich Mobile App

1.80.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
not
```

### Your .env content

```Shell
not
```

### Reproduction steps

```bash
1.
2.
3.
...
```

**Comment by alextran1502** [comment]
It can be now, please double check and reopen if not
